index.html

```diff
@@ -0,0 +1,285 @@
+    </section>
+    <section class="about-me">
+      <div class="about">
+        <div class="contaner5">
+          <h2 class="greads">
+            About <br />
+            Myself
+          </h2>
+          <p class="intrudusing">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h2 class="LETS connect">LET’S CONNECT</h2>
+          <ul class="icons_cont">
+            <li>
+              <a href=""><i class="fab fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-linkedin-in"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-angellist"></i></a>
+            </li>
+            <li>
+              <a href=""> <img src="./img/Vector-1.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-github"></i></a>
+            </li>
+          </ul>
+          <button type="button">Get my resume</button>
+        </div>
+        <div class="container4">
+          <div class="last-part">
+            <ul class="ll11">
+              <li>
+                <h3>Languages</h3>
+              </li>
+              <li>
+                <i class="fas fa-angle-down"></i>
+              </li>
+            </ul>
+            <ul class="Languages-1">
+              <li class="language">
+                <img src="./img/js.png" alt="" />
+                <p>JavaScript</p>
+              </li>
+              <li class="language">
+                <img src="./img/html.png" alt="" />
+                <p>HTML</p>
+              </li>
+              <li class="language">
+                <img src="./img/css.png" alt="" />
+                <p>CSS</p>
+              </li>
+            </ul>
+            <ul class="frameworks">
+              <li>
+                <h3>Frameworks</h3>
+              </li>
+              <li>
+                <i class="fas fa-angle-right"></i>
+              </li>
+            </ul>
+            <hr />
+            <ul class="skills">
+              <li>
+                <h3>Skills</h3>
+              </li>
+              <li>
+                <i class="fas fa-angle-right"></i>
+              </li>
+            </ul>
+            <hr />
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <footer id="contact">
+      <div class="form-container">
+        <div class="form-title">
+          <h3>Contact Me</h3>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form action="https://formspree.io/f/mrgrvdkl" method="post">
+          <ul class="ulclass">
```

Comment:
- [ ] **Required:** As much as possible your design should be similar to your chosen template as stated earlier.

Your Design | Figma Template 
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/150540938-f1d3fce6-681d-49c2-a65e-385e3fa3e961.png) | ![image](https://user-images.githubusercontent.com/27709832/150541031-e09d8ec8-bc87-4fd8-8de7-ea7e34dbd036.png)